Try adding a process.on('uncaughException' ?